security - (JWT, password hashing)

ivor11 opened the pull request:
add jwt token auth
hash passwords in DB
allow multiple user logins
remove logout() as invalidating jwt can be done by deleting token

rohansikder reviewed (changes requested):
Can you update the postman collection to make it work with the new changes. Last night I updated them to work, Just was having issues with Booking Querys

ivor11 commented:
> Can you update the postman collection to make it work with the new changes. Last night I updated them to work, Just was having issues with Booking Querys

I've updated the postman collection, ci should run successfully now

rohansikder commented:
> > Can you update the postman collection to make it work with the new changes. Last night I updated them to work, Just was having issues with Booking Querys
> 
> 
> 
> I've updated the postman collection, ci should run successfully now

Had a glance through this through the phone, Looks good feel free to merge
